scene/2d/tile_map.h

```diff
@@ -60,7 +61,7 @@ class TileMap : public TileMapLayerGroup {
 	friend class TileSetPlugin;
 
 	// A compatibility enum to specify how is the data if formatted.
-	mutable TileMapDataFormat format = TileMapDataFormat::FORMAT_3;
+	mutable TileMapDataFormat format = TileMapDataFormat::TILE_MAP_DATA_FORMAT_3;
```

Comment:
I prefer the change to the default format being explicit, so that it is visible in the change log.
Also it would require a cast like `(TileMapDataFormat) (TILE_MAP_DATA_FORMAT_MAX - 1)`